tests/ltp/patches/getxattr03.patch

```diff
@@ -0,0 +1,64 @@
+diff --git a/testcases/kernel/syscalls/getxattr/getxattr03.c b/testcases/kernel/syscalls/getxattr/getxattr03.c
+index b6ea14287..f8b651c34 100644
+--- a/testcases/kernel/syscalls/getxattr/getxattr03.c
++++ b/testcases/kernel/syscalls/getxattr/getxattr03.c
+@@ -27,6 +27,10 @@
+  * the current size of the named extended attribute.
+  */
+ 
++/* Currently xattr is not enabled while mounting root file system. Patch is
++ * to mount root file system with xattr enabled and then use it for the test.
++*/
++
+ #include "config.h"
+ #include <sys/types.h>
+ #include <sys/stat.h>
+@@ -39,7 +43,7 @@
+ #include <stdlib.h>
+ #include <string.h>
+ #ifdef HAVE_SYS_XATTR_H
+-# include <sys/xattr.h>
++#include <sys/xattr.h>
```

Comment:
not needed

Reply:
@letmaik I asked this to be fixed since #include looks more proper than # include. Don't you aggree? Do we really need to take it back?

Reply:
@letmaik . I have updated as per comments.